sql/moz-fx-data-shared-prod/firefox_desktop_derived/desktop_dau_distribution_id_history_v1/query.sql

```diff
@@ -0,0 +1,49 @@
+-- The desktop baseline ping has historically not had the `distribution_id` field required for us to be able to filter
+-- MozillaOnline clients from the KPI counts. We anticipate that this field will be added to the baseline ping for the
+-- launch of 136.
+-- This table creates a time series of distribution_ids from legacy (main) pings for desktop DAU based on the baseline
+-- ping.
+-- This would enable us to filter MozillaOnline (distribution_id='MozillaOnline') clients from the KPI purposes
+WITH baseline_users AS (
+  SELECT
+    submission_date,
+    client_id,
+    legacy_telemetry_client_id,
+    (browser_engagement_uri_count > 0)
+    AND (browser_engagement_active_ticks > 0) AS is_dau_baseline,
```

Comment:
Wait, do we need this here? I thought we agreed that this table should only contain submission_date, legacy_telemetry_client_id, client_id, and distribution_id?

Reply:
@kik-kik is right, the minimal version of this table only contains those three columns. ill add suggestions